src/commands/org/login/jwt.ts

```diff
@@ -49,7 +49,7 @@ export default class LoginJwt extends AuthBaseCommand<AuthFields> {
       summary: commonMessages.getMessage('flags.instance-url.summary'),
       description: commonMessages.getMessage('flags.instance-url.description'),
       deprecateAliases: true,
-      aliases: ['instanceurl'],
+      aliases: ['instanceurl', '-l'],
```

Comment:
You don't need to put a hyphen before short char aliases... unless you need `--l` to work for some reason